### DISCUSSION TITLE
All the websites UI are broken while I am using Dark Reader- potential bug?

### DISCUSSION THREAD

**Discussion by yorgos-s** [darkreader/darkreader#12003] [General]
I'm not entirely sure what the issue is, but while I was typing in Google Docs, suddenly (perhaps I accidentally activated a shortcut?), the user interface of all sites became somewhat disrupted. I was using Dark Reader in Chrome on Windows 10 (in static mode). It's as if there's an additional layer obscuring some elements that have loaded, and I can't seem to remove it. I've tried restarting Dark Reader, but this had no effect. Certain sites, like ChatGPT, become unusable with Dark Reader activated. It's worth noting that I've never had any problems with Dark Reader before; it's been the most stable dark mode extension I've used.

*DEll Inspiron5577, i7 2.8GHZ,  Nvidia GeForce GTX 1050 TI 4GB

Please see the atached image  w/    chatGPT, Google Docs, Youtube channel

                                                          https://imgur.com/a/v3nNOE9

**Comment by alexanderby** [comment]
Could you please tell the version of your browser?

**Comment by yorgos-s** [comment]
Hi there

Google Chrome  -  Version 120.0.6099.71 (Official Build) (64-bit)
  - on Windows OS  10.0.19045  Build 19045

Please notice that this is a problem of the static mode and only.
Its like there is a layer upon everything.




________________________________

**Comment by alexanderby** [comment]
Oh, the Static mode, it's pretty abandoned and there are plans for its full rework. Let me see what could go wrong...

**Comment by yorgos-s** [comment]
To be honest I have found a workaround using the dynamic mode - which I used in the past - so there is no issue on my end.
I switched to static mode 1 year ago because of the google docs mess (google docs is my primary target).

Thanks for all your hard work, Alexander.




________________________________
